### PR TITLE
Remove debugging comments

### PR DESCRIPTION
**Pull request by adrien-zinger**
cleaning the code :broom:

### CODE REVIEW

**Review by AureliaDolo — DISMISSED** [review]
indeed

**Comment by AureliaDolo** [comment]
bors merge

**Comment by yvan-sraka** [comment]
bors merge

**Comment by yvan-sraka** [comment]
bors merge

**Review by AureliaDolo — APPROVED** [review]

**Comment by AureliaDolo** [comment]
bors merge

File: massa-node/src/settings.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ lazy_static::lazy_static! {
     pub static ref SETTINGS: Settings = {
         let mut settings = config::Config::default();
         let config_path = std::env::var("MASSA_CONFIG_PATH").unwrap_or_else(|_| "base_config/config.toml".to_string());
-        settings.merge(config::File::with_name(&config_path)).unwrap_or_else(|_| panic!("failed to read {} config.... {}", config_path, std::env::current_dir().unwrap().as_path().to_str().unwrap()));
```
Comment (yvan-sraka) [inline massa-node/src/settings.rs:20]: @adrien-zinger I feel like most of the changes of this PR were already done by me here https://github.com/massalabs/massa/pull/1848/files#diff-358b9e753ef58398a8d42a5690fb62569b34b39318406e70b51c99fafad8071bL35

**Review by yvan-sraka — APPROVED** [review]

**Comment by adrien-zinger** [comment]
duplicated

**Pull request closed without merging**